packages/@guardian/source-react-components-development-kitchen/src/toggle-switch/ToggleSwitch.tsx

```diff
@@ -39,12 +46,24 @@ export interface ToggleSwitchProps extends Props {
 	 */
 	labelPosition?: LabelPosition;
 	/**
-	 * Whether the toggle has a tooltip.
+	 * Displays a border above the toggle switch label and toggle.
+	 */
+	labelBorder?: boolean;
+	/**
+	 * If JS is not available the toggle is not clickable and shows a tooltip.
 	 * The default is false.
 	 */
-	tooltip?: boolean;
+	noJs?: boolean;
```

Comment:
could this name be more descriptive perhaps? maybe something like `showNoJavascriptTooltip`?

Reply:
The tooltip is being binned imminently in favour of a fallback so really all this prop is for is for the platform to specify whether or not JS is available, it might be that down the line this changes but i feel like for now it's suitably generic :)